mmseg/models/losses/tversky_loss.py

```diff
@@ -75,6 +78,9 @@ class TverskyLoss(nn.Module):
         beta (float, in [0, 1]):
             The coefficient of false negatives. Default: 0.7.
             Note: alpha + beta = 1.
+        gamma (float, in [1, 3]): The focal term. When `gamma` > 1, the loss
```

Comment:
We may add assertion below to ensure `1.0 < gamma < 3.0`, so do unit test.

Reply:
The [1, 3] range was mentioned in the paper, although they do not mention why `gamma` should be in this range. Theoretically, `gamma` can be any real number. Practically, a small `gamma` (e.g. < 2) works better. Perhaps we should remove the requirement for `gamma` in [1, 3] and let the user to decide?

Reply:
Beside, I also think `alpha` and `beta` in the original `TverskyLoss` class does not need to satisfy `assert alpha + beta == 1.0`. Floating point equality is dangerous and according the definition of the [Tversky index](https://en.wikipedia.org/wiki/Tversky_index), the only requirement for `alpha` and `beta` is that they are non-negative numbers.